distributions/aws/examples/cognito-rds-s3/README.md

```diff
@@ -8,16 +8,9 @@ Follow the pre-requisites section from [this guide](../rds-s3/README.md#1-prereq
 1. Clone the repo
 1. Create an EKS cluster and
 1. Create S3 Bucket
-
-## Configure Kubeflow Pipelines for RDS and S3
```

Comment:
user still need to follow this in addition to prereqs. maybe you can change the section name to
`Configure RDS endpoint and S3 bucket name for Kubeflow Pipelines` to make it clear